changelog.md

```diff
@@ -2,10 +2,15 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+* EVM inbounds support multiple calls inside same tx. EVM Gateway contracts must be upgraded before node upgrade, and additional action fee should be set.
```

Comment:
```suggestion
* EVM inbounds support multiple calls inside same tx. EVM Gateway contracts must be upgraded before node upgrade, and an additional action fee should be set.
```
Should we call this deposit fee instead btw?

Reply:
in case of a `call` it is not deposit so action fee seemed more correct to me, wdyt?